cmd/juju/application/deployer/bundlehandler_test.go

```diff
@@ -64,15 +64,15 @@ func (s *BundleDeployRepositorySuite) SetUpTest(_ *gc.C) {
 	s.deployArgs = make(map[string]application.DeployArgs)
 	s.output = bytes.NewBuffer([]byte{})
 
-	// TODO: remove this patch once we removed all the old series from tests in current package.
-	s.PatchValue(&SupportedJujuSeries,
-		func(time.Time, string, string) (set.Strings, error) {
-			return set.NewStrings(
-				"centos7", "centos8", "centos9", "genericlinux", "kubernetes", "opensuseleap",
-				"jammy", "focal", "bionic", "xenial",
-			), nil
-		},
-	)
+	// TODO: remove this patch once we removed all the old bases from tests in current package.
+	s.PatchValue(&corecharm.SupportedJujuBases, func(time.Time, series.Base, string) ([]series.Base, error) {
```

Comment:
There should be a local SupportedJujuBases for this package to replace the local SupportedJujuSeries.

Reply:
That would be possible, but would require injecting them into the base selector. Which I did consider, but thought this would be lower impact